### ISSUE TITLE
ldap_person Object Improvements

### ISSUE THREAD

**Issue by mikeradka** [ocsf/ocsf-schema#848]
Some input we gathered regarding the new `ldap_person` object (which replaces the `Person` profile via #813:

- There are a few required attributes from the 3 LDAP classes we would want to make sure we have covered, and the attribute names should match:
  - `ldap_dn` should be shortened to `dn`
  - `ldap_cn` should be shortened to `cn`
  - LDAP (`sn`) surname is not in the current object. Since it is a key LDAP object, it should be added.
 
- `employee_uid` is `uid` in LDAP (OCSF uses uid too, maybe that can be shortened within the object, or have our object extend _entity so we pick up name and uid)
- `given_name` which is `givenName` in LDAP
  - on this one, I'm not sure switching to camel-case would be a good practice to do, but adding it so we do not forget it was raised.

**Comment by mtharp0** [comment]
I agree with `dn`, `cn`, and `sn`. I think we need to keep the lowercase `given_name`. If we extent `_entity` I think we pick up the constraint for `name` or `uid`. So, we would want to open up the constraints if we do that.

**Comment by mikeradka** [comment]
Following thorough deliberation in our System Activity syncs, we have reached the conclusion that while shortening the attribute names may seem desirable, implementing such changes would constitute a breaking modification. Alternatively, one might consider deprecating the current attributes in favor of newly abbreviated ones. However, given the importance of exercising discretion with deprecations, our recommended approach is to maintain the existing attributes in their current form at this juncture.
